### ISSUE TITLE
Not receiving data in terminal 

### ISSUE THREAD

**Issue by salivo** [Serial-Studio/Serial-Studio#172]
Hello,
there is no data in the Serial Studio console, but there is data in the Arduino monitor
It seams like  #86.
When i close and reopen connection it can give random data

### Screenshots
![image](https://github.com/Serial-Studio/Serial-Studio/assets/71031134/af83b6fe-0e5c-42dc-9fbf-113a6605c434)
Screenshot after several clicks on connect/disconnect
![image](https://github.com/Serial-Studio/Serial-Studio/assets/71031134/b8bb0684-863f-4b84-9ecc-877dfccca040)

## Simple code in ESP-IDF for ESP32
```c 
#include <stdio.h>

void app_main(void)
{
	while (1)
	{
		printf("*/123,123,1234/*\n");
	}
	
}
```

**Comment by salivo** [comment]
### Update
these are not completely random data, I see fragments of correct data in them
![isue](https://github.com/Serial-Studio/Serial-Studio/assets/71031134/abb56484-5408-48f6-b9b5-d64d7d6e2c2c)

**Comment by salivo** [comment]
I think I found the problem, it's in RST and DRT, can I manually control it?

**Comment by Sarah26-10** [comment]
I have the problem with arduino Portenta H7 Machine control. If i connect this device i receive no data at all.
What can i do?

When i connect a arduino nano every everything is working.
:)

**Comment by recursivenomad** [comment]
Could this be related to #124 ([comment](https://github.com/Serial-Studio/Serial-Studio/issues/124#issuecomment-2167877139))?  I have been unable to get serial data from my Raspberry Pi Pico.

TLDR, are you on Windows and does the serial stream work if you first open it in the Arduino monitor, then restart Serial Studio?

**Comment by salivo** [comment]
Yes, Thank you, I resolve the problem, just, manually disconnect DTR and RTS pin from my TTL convertor. And it works fine. I wait for update #198 with manual control DTR .

**Comment by alex-spataru** [comment]
Again, thanks to @recursivenomad for fixing the issue. The latest build adds a checkbox that allows users to send the DTR (Data Terminal Ready) signal upon connecting to a serial port device:

<img width="1204" alt="Screenshot 2024-08-05 at 3 40 40 p m" src="https://github.com/user-attachments/assets/9e9a3019-3bf8-4ec6-8a4d-5c14c73b71b9">

Please let me know if the latest [continuous](https://github.com/Serial-Studio/Serial-Studio/releases/tag/continuous) build fixes this issue to draft a new release.
